codalab_service.py

```diff
@@ -366,6 +366,13 @@ def has_callable_default(self):
     ),
     # Public workers
     CodalabArg(name='public_workers', help='Comma-separated list of worker ids to monitor'),
+    CodalabArg(
+        name='recaptcha_site_key',
+        env_var='REACT_APP_RECAPTCHA_SITE_KEY',
```

Comment:
Let's rename this REACT_APP_CODALAB_RECAPTCHA_SITE_KEY instead

Reply:
Can we make all environment variables start with `$CODALAB_` and not have to have two?

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/
Looks like for create-react-app, it must start with `REACT_APP_`

Reply:
> 
> 
> Let's rename this REACT_APP_CODALAB_RECAPTCHA_SITE_KEY instead. Also, let's not add a custom env_var here -- instead, in the mapping in docker-compose.yml, we can do REACT_APP_CODALAB_RECAPTCHA_SITE_KEY=${CODALAB_RECAPTCHA_SITE_KEY}

I wanted to have default keys so the pages can run out of the box, are there any other ways to do it? Otherwise developers cannot use the signup page out of the box.

Reply:
Default keys are fine. My suggestion was that we could still make all environment variables to codalab_service.py start with `$CODALAB_` (as @percyliang suggested) by adding the line `REACT_APP_CODALAB_RECAPTCHA_SITE_KEY=${CODALAB_RECAPTCHA_SITE_KEY}` in docker-compose.yml. 

This way, we're essentially hiding the implementation logic (the fact that environment variables in the frontend must start with `REACT_APP_`) and presenting a consistent interface to the user who is actually running codalab_service.py (the interface being that "all environment variables to codalab_service.py must start with `$CODALAB_`").

Reply:
@epicfaace  I see what you mean. I have updated the code.